python-avocado.spec

```diff
@@ -249,8 +249,6 @@ AVOCADO_CHECK_LEVEL=0 selftests/run
 %{_bindir}/avocado-rest-client
 %{_mandir}/man1/avocado.1.gz
 %{_mandir}/man1/avocado-rest-client.1.gz
-%{_docdir}/avocado/avocado.rst
```

Comment:
Hmm, how do they get there? Anyway I'll double-check in functional review and if that's true it's a nice simplification...

Reply:
Typo in commit message: I meant: "There's little to no point..."

Reply:
Oh, yes, one should read it carefully. Basically the `man` pages are and will be included, but the `rst` "sources" won't, right?